docs/source/details/backendconfig.rst

```diff
@@ -35,6 +35,7 @@ Parameters that are directly passed through to an external library and not inter
 
 The configuration string may refer to the complete ``openPMD::Series`` or may additionally be specified per ``openPMD::Dataset``, passed in the respective constructors.
 This reflects the fact that certain backend-specific parameters may refer to the whole Series (such as storage engines and their parameters) and others refer to actual datasets (such as compression).
+Dataset-specific configurations are (currently) only available during dataset creation, but not when reading datasets.
```

Comment:
We should try to solve this until the next release, so this note is temporary.
Solving this exceeds the scope of this PR as there are some design considerations involved.